--- config.orig	2018-05-21 15:03:13.783227489 +0800
+++ config	2018-05-21 16:02:29.162024219 +0800
@@ -175,3 +175,106 @@
 
 # default config
 # =============================================================================
+default_orientation horizontal
+new_window pixel 0
+mouse_warping none
+
+
+# i3bar to display a workspace bar
+# -----------------------------------------------------------------------------
+bar {
+
+    mode hide
+    hidden_state hide
+    modifier $mod
+
+    position bottom
+    strip_workspace_numbers no
+
+    colors {
+        separator  #666666
+        background #222222
+        statusline #dddddd
+        focused_workspace  #0088CC #0088CC #ffffff
+        active_workspace   #333333 #333333 #ffffff
+        inactive_workspace #333333 #333333 #888888
+        urgent_workspace   #2f343a #900000 #ffffff
+    }
+
+    status_command i3status
+}
+
+
+# workspace output
+# -----------------------------------------------------------------------------
+set $left_monitor DVI-D-0
+set $right_monitor VGA-0
+
+workspace 1 output $left_monitor
+workspace 3 output $left_monitor
+workspace 5 output $left_monitor
+workspace 7 output $left_monitor
+workspace 9 output $left_monitor
+
+workspace 2 output $right_monitor
+workspace 4 output $right_monitor
+workspace 6 output $right_monitor
+workspace 8 output $right_monitor
+workspace 10 output $right_monitor
+
+
+# for window
+# -----------------------------------------------------------------------------
+# automatically switch to a workspace when opening an application assigned to it
+for_window[class=".*"] focus
+# set urxvt window's border
+for_window [class="(?i)urxvt"] border pixel 4
+
+
+# assignment
+# -----------------------------------------------------------------------------
+assign [class="(?i)google-chrome"] $ws1
+assign [class="(?i)urxvt"] $ws2
+assign [class="(?i)Foxit Reader"] $ws3
+assign [class="shadowsocks-qt5"] $ws0
+assign [class="netease-cloud-music"] $ws0
+
+
+# key bind
+# -----------------------------------------------------------------------------
+# window lock
+bindsym $mod+z exec "i3lock -i ~/Pictures/wallpapers/lock.png -c 000000"
+
+# TODO: need to set Master/Headphone/Front
+bindsym XF86AudioRaiseVolume exec amixer -q set Master 2dB+ unmute
+bindsym XF86AudioLowerVolume exec amixer -q set Master 2dB- unmute
+#bindsym XF86AudioMute exec amixer -q set Master toggle
+
+
+# startup programe
+# -----------------------------------------------------------------------------
+exec --no-startup-id feh --bg-fill ~/Pictures/wallpapers/wallpaper.jpg
+exec "~/Tools/shadowsocks/Shadowsocks-Qt5-3.0.0-x86_64.AppImage"
+
+# default "Startup" workspace
+exec --no-startup-id i3-msg rename workspace 1 to $ws1
+exec --no-startup-id i3-msg rename workspace 2 to $ws2
+
+
+# customized mode
+# -----------------------------------------------------------------------------
+mode "application" {
+    bindsym $mod+l exec notify-send -t 10 "\nc chrome\ne netease-cloud-music\nf FoxitReader\nn nautilus\ns shadowsocks"
+
+    bindsym c exec google-chrome; mode "default"
+    bindsym e exec netease-cloud-music; mode "default"
+    bindsym f exec FoxitReader; mode "default"
+    bindsym n exec nautilus; mode "default"
+    bindsym s exec "~/Tools/shadowsocks/Shadowsocks-Qt5-3.0.0-x86_64.AppImage"; mode "default"
+
+    bindsym Return mode "default"
+    bindsym Escape mode "default"
+    bindsym $mod+p mode "default"
+}
+
+bindsym $mod+p mode "application"
