--- config.def.h	2019-04-11 12:45:48.026335693 +0800
+++ config.h	2019-04-11 13:29:10.322938728 +0800
@@ -62,6 +62,21 @@ static const char unknown_str[] = "n/a";
  * wifi_essid          WiFi ESSID                      interface name (wlan0)
  */
 static const struct arg args[] = {
-	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	/* function         format          argument */
+    { cpu_freq,         " C: %6sHz ",   NULL },
+    { temp,             "%2s°C ",       "/sys/class/thermal/thermal_zone3/temp" },
+    { load_avg,         " %s | ",       NULL },
+    { ram_used,         "M: %3sB ",     NULL },
+    { ram_perc,         "%2s%% | ",     NULL },
+    { swap_used,        "S: %sB ",      NULL },
+    { swap_perc,        "%2s%% | ",     NULL },
+    { disk_used,        "D: %3sB ",     "/"  },
+    { disk_perc,        "%2s%% | ",     "/"  },
+    { battery_perc,     "B: %2s%% ",    "BAT1" },
+    { battery_remaining,"%6s | ",       "BAT1" },
+    { wifi_essid,       "W: %s ",       "wlp3s0"},
+    { wifi_perc,        "%3s%% | ",     "wlp3s0"},
+    { netspeed_rx,      "RX: %8s ",     "wlp3s0"},
+    { netspeed_tx,      "TX: %8s | ",   "wlp3s0"},
+    { datetime,         "%s",           "%F %T %a" },
 };
